modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -251,6 +251,54 @@ public void testGetParametersShouldUseFileEntryTitle() {
 		Assert.assertEquals("New Title", parameters.get("fileEntryTitle"));
 	}
 
+	@Test
+	public void testGetParametersWithNullGroupShouldContainItemSelectorURL() {
+		PowerMockito.when(
+			_groupLocalService.fetchGroup(_GROUP_ID)
+		).thenReturn(
+			null
+		);
+
+		PowerMockito.when(
+			_itemSelector.getItemSelectorURL(
+				Mockito.eq(_requestBackedPortletURLFactory),
+				Mockito.eq(_scopeGroup), Mockito.eq(_GROUP_ID),
+				Mockito.eq(_PORTLET_NAMESPACE + "selectDocumentLibrary"),
+				Mockito.any(FileItemSelectorCriterion.class))
+		).thenReturn(
+			new MockLiferayPortletURL()
+		);
```

Comment:
@carolmariaabb, maybe Brian will complain about this mock be here on the test method instead of have a dedicated method to mock the dependencies, what do you think?

Reply:
you're right @natocesarrego, let me adjust this